private/Producers/FbxProducer/FbxProducerImpl.cpp

```diff
@@ -1392,26 +1397,20 @@ void FbxProducerImpl::ImportSkeletonBones(fbxsdk::FbxScene* pScene, const std::v
 		}
 
 		// Get local transform materix for link bone.
-		fbxsdk::FbxVector4 localLinkT;
-		fbxsdk::FbxQuaternion localLinkQ;
-		fbxsdk::FbxVector4 localLinkS;
+		fbxsdk::FbxAMatrix transform;
 		if (boneIndex != rootBoneIndex)
 		{
 			fbxsdk::FbxAMatrix transform = globalTransformPerLinkBone[cdBoneParentID].Inverse() * globalTransformPerLinkBone[cdBoneID];
-			localLinkT = transform.GetT();
-			localLinkQ = transform.GetQ();
-			localLinkS = transform.GetS();
+			
 		}
 		else
 		{
 			// RootBone is already in global coordinate system.
-			const auto& rootTransform = globalTransformPerLinkBone[cdBoneID];
-			localLinkT = rootTransform.GetT();
-			localLinkQ = rootTransform.GetQ();
-			localLinkS = rootTransform.GetS();
+			const auto& transform = globalTransformPerLinkBone[cdBoneID];
+		
```

Comment:
Defined variable `transform` twice.
Also, you should consider `boneIndex != rootBoneIndex`. Please have a check if `transform` is an identity matrix.
```suggestion
			transform = globalTransformPerLinkBone[cdBoneID];
		
```